app/site/index.liquid

```diff
@@ -2,57 +2,66 @@
+    </p>
+    <p>
+      {{ site.description }}
+    </p>
+  </section>
+  <ul class="usa-card-group-home">
     <li class="usa-card tablet:grid-col-6">
-      <div class="usa-card__container">
-        <div class="usa-card__header">
-          <h2 class="usa-card__heading">
-            <svg class="usa-icon" aria-labelledby="organizations" role="img">
-              <title id="organizations-category"> Organizations </title>
-              {% lucide "building-2" %}
-            </svg>
-            <a href="{{ "/organizations/" | url}}" class="text-no-underline"
-              >Organizations
-              <span class="chevrons-right">
-                <svg class="usa-icon usa-icon--size-2" aria-hidden="true" focusable="false" role="img">
-                  <use xlink:href= {{ "/assets/img/sprite.svg#navigate_far_next" | url }}></use>
+      <div class="usa-card-outer">
+        <div class="usa-card-middle">
+          <a href="{{ "/organizations/" | url}}" class="href-home">
+            <div class="usa-card__container">
+              <div class="home-icon-container">
+                <svg class="usa-icon--size-5" aria-hidden="true" focusable="false" role="img">
+                  <use xlink:href="/assets/img/sprite.svg#groups"></use>
                 </svg>
-              </span>
-            </a>
-          </h2>
-        </div>
-        <div class="usa-card__body">
-          <p>View Metrics on our GitHub Organizations</p>
+              </div>
+              <div class="usa-card__header">
+                <h2 class="usa-card__heading-home">
+                  Organizations
+                </h2>
+                <div class="usa-card__body">
+                  <p>View Metrics on our GitHub Organizations</p>
+                </div>
+              </div>
+            </div>
+          </a>
         </div>
       </div>
     </li>
     <li class="usa-card tablet:grid-col-6">
-      <div class="usa-card__container">
-        <div class="usa-card__header">
-          <h2 class="usa-card__heading">
-            <svg class="usa-icon" aria-labelledby="projects" role="img">
-              <title id="projects-category"> Projects </title>
-              {% lucide "folder-git-2" %}
-            </svg>
-            <a href="{{ "/projects/" | url}}" class="text-no-underline"
-              >Projects
-              <span class="chevrons-right">
-                <svg class="usa-icon usa-icon--size-2" aria-hidden="true" focusable="false" role="img">
-                  <use xlink:href={{ "/assets/img/sprite.svg#navigate_far_next" | url }} ></use>
+      <div class="usa-card-outer">
+        <div class="usa-card-middle">
+          <a href="{{ "/projects/" | url}}" class="href-home">
+            <div class="usa-card__container">
+              <div class="home-icon-container">
+                <svg class="usa-icon--size-5" aria-hidden="true" focusable="false" role="img">
+                  <use xlink:href="/assets/img/sprite.svg#folder_open"></use>
```

Comment:
This icon is not showing up when I run `npm run prod`. This is not being able to find the path to this image since due to prod prepending "/metrics" to every path. To fix, I think the 11ty url filter needs to be used:
`{{"/assets/img/usa-icons/close.svg" | url }}`
https://www.11ty.dev/docs/filters/url/
<img width="1381" alt="Screenshot 2024-12-02 at 10 34 55 AM" src="https://github.com/user-attachments/assets/8dda0f0a-839c-4c3c-9dda-f9ef780985fb">